src/test/resources/org/broadinstitute/barclay/help/expected/HelpDoclet/org_broadinstitute_barclay_help_testinputs_TestArgumentContainer.html

```diff
@@ -323,6 +330,28 @@ <h3><a name="--deprecatedString">--deprecatedString </a>
 				&nbsp;<span class="label label-warning">∞ ] ]</span>
 		</p>
 		<hr style="border-bottom: dotted 1px #C0C0C0;" />
+		<h3><a name="--enumCollection">--enumCollection </a>
+			
+		</h3>
+		<p class="args">
+			<b>Undocumented option</b><br />
+			
+		</p>
+			<p>
+				The --enumCollection argument is an enumerated type (List[TestEnum]), which can have one of the following values:
```

Comment:
This doc might be slightly confusing, since the collection can have more than 1 of the possible values.  This is an improvement, and I'm not sure how much effort we want to put into this very minor feature, but it might be nice to make the documentation aware that its' a collection.